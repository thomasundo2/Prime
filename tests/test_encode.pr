int main()
{
    string in;
    lint encoded;
    in = "HelloWorld";
    prints(in);
    encode(encoded, in);
    printl(encoded);
    prints(decode(encoded));
    return 0;
}