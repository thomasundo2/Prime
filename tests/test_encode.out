HelloWorld
72101108108111087111114108100
HelloWorld
